src/resultset/resultset.h

```diff
@@ -20,17 +20,17 @@ typedef struct {
 	RedisModuleCtx *ctx;            /* Redis context. */
 	GraphContext *gc;               /* Context used for mapping attribute strings and IDs */
 	uint column_count;              /* Number of columns in result set. */
-	bool compact;                   /* Whether records should be returned in compact form. */
 	bool header_emitted;            /* Whether a header row has been issued to the user. */
 	const char **columns;           /* Field names for each column of results. */
 	uint *columns_record_map;       /* Mapping between column name and record index.*/
 	size_t recordCount;             /* Number of records introduced. */
 	double timer[2];                /* Query runtime tracker. */
 	ResultSetStatistics stats;      /* ResultSet statistics. */
+    ResultSetFormatterType format;  /* Result-set format; compact/verbose/nop. */
```

Comment:
I'm surprised that we can't remove this member and check against the `ResultSetFormatter` directly at https://github.com/RedisGraph/RedisGraph/pull/1004/files#diff-0ba9657f80a4ecf3a1b11bda5e1df1a2R154

But a cursory test of that modification doesn't work, and this is a trivial point, so no big deal!